### PR TITLE
kde-builder: add page

### PR DESCRIPTION
**Pull request by Managor**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/kde-builder.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# kde-builder
+
+> Easily build KDE components from its source repositories.
+> Drop-in replacement for kdesrc-build.
```
Comment (spageektti) [inline pages/linux/kde-builder.md:4]:
```suggestion
> Drop-in replacement for `kdesrc-build`.
```

**Review by spageektti — APPROVED** [review]

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution, for now, I would like to only add this suggestion, based on discussion #13441, I will make a few suggestions for the short options soon.

File: pages/linux/kde-builder.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+- Initialize `kde-builder`:
+
+`kde-builder --initial-setup`
+
+- Compile a KDE component and its dependencies from the source:
+
+`kde-builder {{component_name}}`
+
+- Compile a component without updating its local code and without compiling its dependencies:
```
Comment (kbdharun) [inline pages/linux/kde-builder.md:15]:
```suggestion
- Compile a component without updating its local code and without compiling its [D]ependencies:
```

File: pages/linux/kde-builder.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+`kde-builder --no-src --no-include-dependencies {{component_name}}`
+
+- Refresh the build directories before compiling:
```
Comment (kbdharun) [inline pages/linux/kde-builder.md:19]:
```suggestion
- [r]efresh the build directories before compiling:
```

**Review by kbdharun — APPROVED** [review]
Hi, sorry for the delay (had a lot of work the past few days). After adding this the PR should be GTG.

**Pull request merged**